sdk/jest.config.js

```diff
@@ -2,7 +2,7 @@
 module.exports = {
   preset: "ts-jest",
   testEnvironment: "node",
-  testTimeout: 15000,
+  testTimeout: 120000,
```

Comment:
maybe we can change this back now

```suggestion
  testTimeout: 15000,
```